swap_meet/item.py

```diff
@@ -1,2 +1,17 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0, age = 0):
+        if not id:
+            id = uuid.uuid4().int
+        self.id = id
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+    def get_category(self):
+        return f"{self.__class__.__name__}"    
```

Comment:
[nit] `__name__` already returns a string, so you don't need the f string.

```suggestion
        return self.__class__.__name__
```